CONTRIBUTING.md

```diff
@@ -43,7 +43,7 @@ GitHub provides additional document on [forking a repository](https://help.githu
 
 ## Tips for Modifying the Source Code
 
-- Using a fresh virtualenv, install the package via [these instructions](https://autogluon.mxnet.io/install.html).
+- Using a fresh virtualenv, install the package via [these instructions](https://auto.gluon.ai/dev/install.html).
 Be sure to select the *Source* option from the installation preferences, and install the package after cloning this repository via:
 ```
 python setup.py develop
```

Comment:
The install instruction is now wrong? Should it instead be: `./full_install.sh`

Reply:
Removed the install instructions here, now users should just use the link to get the website install instructions.